src/platform/plugins/shared/vis_types/timeseries/public/convert_to_lens/lib/configurations/xy/layers.ts

```diff
@@ -190,6 +186,17 @@ export const getLayers = async (
   }
 };
 
+function getEventIcon(annotationIcon: string | undefined): AvailableAnnotationIcon {
```

Comment:
This piece of logic has been written as TS wasn't happy about the type narrowing in the original ternary code.